lib/modulr/api/services.rb

```diff
@@ -1,11 +1,19 @@
 # frozen_string_literal: true
 
+require_relative "service"
+require_relative "accounts_service"
+require_relative "transfers_service"
+
 module Modulr
   module API
     module Services
       def accounts
         @services[:accounts] ||= API::AccountsService.new(self)
       end
+
+      def transfers
```

Comment:
IMO, we should not be consistent between API clients. Clients should represent the much they can the provider API.

Reply:
ok, payments instead is the choosen name 👍🏻